features/net/network-socket/NetworkStack.cpp

```diff
@@ -20,11 +20,29 @@
 #include "stddef.h"
 #include <new>
 
+static bool stackHasIPV4(NetworkStack *stack)
```

Comment:
Please stick with the existing naming convention: `stack_has_ipv4`
https://docs.mbed.com/docs/getting-started-mbed-os/en/latest/Full_Guide/Code_Style/